src/daft-core/src/series/ops/list.rs

```diff
@@ -177,6 +178,109 @@ impl Series {
         }
     }
 
+    /// Given a series of `List` or `FixedSizeList`, return a boolean series
+    /// indicating whether the list contains a value.
+    ///
+    /// The "contains" series that is being checked must be of length 1 or of
+    /// the same length as the series being checked against.
+    ///
+    /// If it is of length 1, it will be broadcasted to the length of the
+    /// series being checked against. This is useful to see if the lists in the
+    /// original series contain a singular value.
+    ///
+    /// If it is of the same length as the series being checked against, each
+    /// value in the "contains" list will be checked against the corresponding
+    /// value in the original list.
+    ///
+    /// # Example
+    /// ```txt
+    /// original: [[1,2,3], [4,5,6], [1,3,5]]
+    /// contains: 1
+    /// result: [true, false, true]
+    /// ```
+    ///
+    /// In the above case, "contains" will be broadcasted to `[1, 1, 1]`.
+    ///
+    /// ```txt
+    /// original: [[1,2,3], [4,5,6], [1,3,5]]
+    /// contains: [5, 3, 1]
+    /// result: [false, false, true]
+    /// ```
+    pub fn list_contains(&self, contains: &Self) -> DaftResult<Self> {
+        let self_length = self.len();
+        let contains = match contains.len() {
+            1 => contains.broadcast(self_length)?,
```

Comment:
Can you use `std::iter::repeat` instead of broadcast? You can do a `.get(0)` after you downcast the series then repeat that. Correct me if im wrong but I think broadcast will make a copy.